pkg/csilvm/arbiter.go

```diff
@@ -0,0 +1,422 @@
+package csilvm
+
+import (
+	"context"
+	"encoding/json"
+	"sort"
+	"sync"
+
+	"github.com/container-storage-interface/spec/lib/go/csi/v0"
+	"github.com/mesosphere/csilvm/pkg/singleflight"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+var _ = csi.IdentityServer(&identityArbiter{}) // sanity check
+
+type RPCCounters struct {
+	IssuedCall func() int64 // callback useful for testing
```

Comment:
The idea here is to wire these hooks to a metrics package at some point. Meanwhile, they're also useful for testing.